src/TuDa.CIMS.Shared/Params/UserParams.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace TuDa.CIMS.Shared.Params;
+
+/// <summary>
+/// A class that represents the parameters for pagination.
+/// </summary>
+public class UserParams
```

Comment:
🔧 🌱 I would rename this to a more declarative name, something like `AssetItemPaginationQueryParams`, so we can use multiple query parameters